definitions/npm/react-native-onesignal_v3.x.x/flow_v0.69.0-/test_react-native-onesignal.js

```diff
@@ -0,0 +1,200 @@
+// @flow
+import OneSignal, {
+  type PermissionSubscriptionState
+} from "react-native-onesignal";
+
+const simpleHandler = () => {};
+
+OneSignal.addEventListener("received", simpleHandler);
```

Comment:
would you mind wrapping test cases inside `describe` and `it` for better readability and maintainability? see https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope for details.